gap/PolygonalComplexes/graphs.gd

```diff
@@ -1099,3 +1099,76 @@ DeclareOperation( "OnVertexEdgePaths",
 DeclareOperation( "OnEdgeFacePaths", 
     [ IsEdgeFacePath , IsPerm ] );
 #! @EndGroup OnEdgeFacePaths
+
+#! @BeginGroup ButterflyInsertion
+#! @Description
+#! Performs Butterfly Insertion on a given graph.
+#! ButterflyInsertion ensures symmetric closure on the given digraph.
+#! ButterfyInsertionNC expects an undirected digraph.
+#! The given graph has to fulfill the following properties:
+#! * it is a mutable graph
+#! The given edges have to fulfill the following properties:
+#! * edges are a List with length 2
+#! * edges must be included in the given graph
+#! @Arguments D, edgeA, edgeB
+#! @Returns the updated graph or an error

```

Comment:
Do not write here that an error can be returned. Write this in the text above and explain there when this happens. Also for the other methods.